attendance/index.html

```diff
@@ -0,0 +1,240 @@
+            </button>
+        </div>
+        <div class="modal-body">
+            <div id="modalError" style="color:red"></div>
+            <h2>Meeting Details</h2>
+            <p id="modalMeetingType">Meeting Type: </p>
+            <p id="modalStartTime">Start Time: </p>
+            <p id="modalEndTime">End Time: </p>
+            <p id="modalDescription">Description: </p>
+            <h2>Meeting Attendees</h2>
+            <div id="modalAttendees">
+
+            </div>
+        </div>
+        <div class="modal-footer">
+            <button type="button" class="btn btn-secondary" id="closeModal" data-dismiss="modal">Close</button>
+            <button type="button" class="btn btn-primary submit-meeting" id="submit" data-dismiss="modal">Create Meeting</button>
+        </div>
+        </div>
+    </div>
+    </div>
+
+    
+    {% include footer.html %}
+    {% include scripts.html %}
+    <script src='//cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.js'></script>
+    <script>
+        let convertMeetingTime = (time) => {
+            if(time.includes('AM') || time.includes('12')) {
+                return time.slice(0, 5);
+            } else {
+                let hour = parseInt(time.slice(0, 3));
+                return `${(hour + 12)}:${time.slice(2, 4)}`;
+            }
+        }
+
+        let validateTime = (start, end) =>{
+            return parseInt(start.replace(':', '')) < parseInt(end.replace(':', ''));
+        } 
+
+        $(() => {
+            fetch('http://backend.famnm.club/meeting_type')
+            .then( (response) =>{
+                if (!response.ok) throw Error(response.statusText);
+                return response.json();
+            }).then( (data) => {
+                data.map(type => {
+                    let meetingType = $('<option></option>');
+                    meetingType.text(type.type);
+                    $('#meetingType').append(meetingType);
+                })
+            })
+
+            $('#meetingDate').attr('value', new Date().toISOString().slice(0,10));
+            $('#meetingDate').prop('disabled', true);
+            
+            $('input.timepicker').timepicker({ 
+                timeFormat: 'h:mm p',
+                minTime: '10:00',
+                maxTime: '22:00',
+                startTime: '10:00',
+                defaultTime: '10:00',
+                dropdown: true,
+                interval: 30
+            });
+            
+            $('#uniqname').keypress((e) => {
+                if (e.keyCode == 13)
+                {
+                    let newButton = $('<button></button>');
+                    newButton.attr('type', 'button');
+                    newButton.addClass('btn');
+                    newButton.addClass('btn-outline-secondary');
+                    newButton.addClass('mx-3');
+                    newButton.addClass('my-3');
+                    newButton.addClass('uniqname-btn');
```

Comment:
{nit} chain these together
```suggestion
                    newButton.addClass('btn').addClass('btn-outline-secondary').addClass('mx-3').addClass('my-3').addClass('uniqname-btn');
```
spacing is a bit off in suggestion